### PR TITLE
OCLOMRS-338: Make the 'no concepts' info be on a react table

### PR DESCRIPTION
**Pull request by dessHub**
# JIRA TICKET NAME:
[OCLOMRS-338: Make the 'no concepts' info be on a react table](https://issues.openmrs.org/browse/OCLOMRS-338)

# Summary:
Fixed this issue by getting rid of the normal bootstrap table and display `No concept!` text on `React Table`.

### CODE REVIEW

**Comment by AGMETEOR** [comment]
@dessHub is RenderTable component still being used anywhere in the files?

**Comment by dessHub** [comment]
Not really at @AGMETEOR

**Comment by dessHub** [comment]
Not really at @AGMETEOR

File: src/tests/bulkConcepts/containers/BulkConceptsPage.test.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ describe('Test suite for BulkConceptsPage component', () => {
     const wrapper = mount(<Router>
       <BulkConceptsPage {...props} />
     </Router>);
-    expect(wrapper.find('#emptyConcept').text()).toEqual('No concepts found');
+    expect(wrapper.find('.rt-noData').text()).toEqual('No concept!');
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: Is no concept better than no concepts found?

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: It makes sense. Updating it right away.

Comment (dessHub) [reply src/tests/bulkConcepts/containers/BulkConceptsPage.test.js]: done

**Pull request merged**